examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/setuppayloadscanner/BarcodeFragment.kt

```diff
@@ -36,11 +36,10 @@ import androidx.camera.lifecycle.ProcessCameraProvider
 import androidx.core.content.ContextCompat
 import androidx.core.content.ContextCompat.checkSelfPermission
 import androidx.fragment.app.Fragment
-import chip.setuppayload.SetupPayload
-import chip.setuppayload.SetupPayloadParser
-import chip.setuppayload.SetupPayloadParser.SetupPayloadException
-import chip.setuppayload.SetupPayloadParser.InvalidEntryCodeFormatException
-import chip.setuppayload.SetupPayloadParser.UnrecognizedQrCodeException
+import chip.onboardingpayload.OnboardingPayload
+import chip.onboardingpayload.OnboardingPayloadParser
+import chip.onboardingpayload.OnboardingPayloadParser.OnboardingPayloadException
+import chip.onboardingpayload.OnboardingPayloadParser.UnrecognizedQrCodeException
```

Comment:
Those exceptions are also used by Generator part, I have moved them to chip.onboardingpayload from chip.onboardingpayload.OnboardingPayloadParser, I will update this part after this PR landed

Reply:
I see.